gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -178,7 +178,7 @@ function GM:PlayerStartVoice(ply)
 	if not IsValid(g_VoicePanelList) or not IsValid(client) then return end
 
 	-- There'd be an extra one if voice_loopback is on, so remove it.
-	GAMEMODE:PlayerEndVoice(ply, true)
```

Comment:
This change breaks the gamemode

Reply:
Why? There is no GAMEMODE:PlayerENdVoice, I checked.

Reply:
![image](https://github.com/TTT-2/TTT2/assets/5711436/d7b19cf2-56db-46be-970f-9e002648f993)

Reply:
Thats GM:PlayerEndVoice for which I exchanged this.

Reply:
WHile I dont get it, GAMEMODE is the actual working hook...... I reseted it

Reply:
GAMEMODE is defined at the first pass when the gamemode is loaded, GM is only available after that

Reply:
> GAMEMODE is defined at the first pass when the gamemode is loaded, GM is only available after that

Is this anywhere documented?

Reply:
I'm pretty sure it is tge other was around. GM is defined on first pass, GAMEMODE, when the game is running